.github/workflows/publish_package.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish Package to Github Packages & upload bundle to S3
+on:
+  release:
+    types: [published]
+env:
+  NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+permissions:
+  contents: read
+  packages: write
+jobs:
+  publish_package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://npm.pkg.github.com'
+          # Defaults to the user or organization that owns the workflow file
+          scope: '@octocat'
+
+      - name: Publish the package
+        run: |
+          npm config set '//npm.pkg.github.com/:_authToken' "${{ secrets.GITHUB_TOKEN }}"
+          npm install
+          npm publish
+
+  s3_upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+
+      # Speed up subsequent runs with caching
+      - name: Cache node modules
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install packages
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn
+
+      - name: Build the bundle
+        run: yarn dist
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
+
+      - name: Upload files to S3 with AWS CLI
+        run: |
+          aws s3 cp dist s3://${{ secrets.AWS_S3_BUCKET }}/${{ github.ref_name }} --recursive --acl public-read
```

Comment:
By using a copy to S3 we are loosing some of the benefits we have in other deployment processes, such as:
* Specify cache time per object
* Specify order in which is deployed (in the Shopper app, for instance, the latest object to be deployed is `index.html`)
* Remove old versions

On the other hand, I guess that S3 restricts access to CloudFront (the CDN), so we should not have the public ACL here.

Reply:
It my fault. I suggest @golive do it in that way. 

What do you mean with: 

> so we should not have the public ACL here.

?

Reply:
What do you suggest?
CDN works fine (https://mufasa.sequracdn.com/v1.0.0/sequra-pci.js) and since each version is put in its own directory, those files will never be replaced. Just when we publish a new library version, it will be uploaded in a new dir and the url will be different.

Reply:
@mpucholblasco do we need to do anything? Can we start using the js stored in S3 through the CDN?

Reply:
Hi!
Sorry for lateness.
As commented with @miquelbar and @golive , the idea is to specify a cache header. AWS cli (according to https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html) allows `--cache-control`, so we should set this parameter to something similar to `public,max-age=31536000`.

On the other hand, we should set `--acl private`, as we enforce the usage of the CDN. With this parameter, we allow accessing to S3 directly.

Things related to removing old versions or deploy order are not relevant for this use case.